Docs/source/visualization/ascent.rst

```diff
@@ -176,3 +176,175 @@ When the trigger condition is meet, ``cycle() == 300``, the actions in :code:`tr
+                  z: 0.0
+                point2:
+                  x: 0.0
+                  y: 0.0
+                  z: 0.0
+                normal2:
+                  x: -0.7
+                  y: -0.7
+                  z: 0.0
+        sampled_particles:
+          f1:
+            type: histsampling
+            params:
+              field: particle_electrons_uz
+              bins: 64
+              sample_rate: 0.90
+          f2:
+            type: "clip"
+            params:
+              topology: particle_electrons # particle data
+              multi_plane:
+                point1:
+                  x: 0.0
+                  y: 0.0
+                  z: 0.0
+                normal1:
+                  x: 0.0
+                  y: -1.0
+                  z: 0.0
+                point2:
+                  x: 0.0
+                  y: 0.0
+                  z: 0.0
+                normal2:
+                  x: -0.7
+                  y: -0.7
+                  z: 0.0
+
+    -
+      action: "add_scenes"
+      scenes:
+        scene1:
+          plots:
+            p0:
+              type: "pseudocolor"
+              field: "particle_electrons_uz"
+              pipeline: "sampled_particles"
+            p1:
+              type: "pseudocolor"
+              field: "Ex"
+              pipeline: "clipped_volume"
+          renders:
+            image1:
+              bg_color: [1.0, 1.0, 1.0]
+              fg_color: [0.0, 0.0, 0.0]
+              image_prefix: "test%06d"
+              camera:
+                azimuth: 20
+                elevation: 30
+                zoom: 2.5
+
+There are Ascent Actions Examples `Ascent Actions Examples <https://ascent.readthedocs.io/en/latest/Actions/Examples.html#yaml-examples>`_ for you to play.
+
+3. Run Replay
+-------------------------
+
+
+Replay executables are created in the utilities/replay directory of the Ascent instaation. There are two version of replay: mpi version replay_mpi and serial version replay_ser. What you use depends on the data set. Here we use replay_mpi as example.
+
+The options for replay are:
+
+--root: specifies Blueprint root file to load
+--cycles: specifies a text file containing a list of Blueprint root files to load
+--actions: specifies the name of the actions file to use (default: ascent_actions.json)
+
+Example launches:
+
+.. code-block:: bash
+
+  srun -n 8 ./replay_mpi --root=test.cycle_002000.root --actions=ascent_action.yaml
+  srun -n 8 ./replay_mpi --cycles=warpx_list.txt --actions=ascent_action.yaml
+
+The cycles files list is a text file containing one root file per line:
+
+.. code-block:: bash
+
+  cat warpx_list.txt
+  test.cycle_000200.root
+  test.cycle_000400.root
+  test.cycle_001000.root
+  test.cycle_002000.root
+
+Replay will loop over these files in the order in which they appear in the file.
+
+The following is an image from replay on test.cycle_000400.root with above ascent_action.yaml
+
+
+.. figure:: WarpX_LWFA_400.png
```

Comment:
I fear this file is too large (nearly 0.5MB) to be commited into WarpX directly. I can show you another location to store it later on.